code/modules/transport/tram/tram_floors.dm

```diff
@@ -186,6 +186,7 @@
 	plane = GAME_PLANE
 	obj_flags = BLOCK_Z_OUT_DOWN | BLOCK_Z_OUT_UP
 	appearance_flags = PIXEL_SCALE|KEEP_TOGETHER
+	custom_materials = list(/datum/material/plastic = SHEET_MATERIAL_AMOUNT)
```

Comment:
This needs to have the same amount of mats per unit as `/obj/item/stack/thermoplastic`.